index.js

```diff
@@ -309,15 +309,19 @@ async function analyzeJsFile(content, userPlugins) {
 
   // find translation keys in the syntax tree
   traverse(ast, {
-    // handle this.intl.t('foo') case
     CallExpression({ node }) {
       let { callee } = node;
-      if (callee.type !== 'MemberExpression') return;
-      if (callee.property.type !== 'Identifier') return;
-      if (callee.property.name !== 't') return;
-
       if (node.arguments.length === 0) return;
 
+      if (callee.type === 'MemberExpression') {
+        // handle this.intl.t('foo') case
+        if (callee.property.type !== 'Identifier') return;
+        if (callee.property.name !== 't') return;
+      } else if (callee.type === 'Identifier') {
+        // handle t('foo') case
+        if (callee.name !== 't') return;
+      }
```

Comment:
Hey thanks for this PR, while i dont think there can be any other types, i haven't work on AST stuff for a while so could you just add a else return to make sure nothing doesn't slip through. Then i'll happily merge and get this released